elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -111,25 +111,14 @@ class PfeContentSet extends PFElement {
     return this.parentNode ? this.parentNode.offsetWidth > breakpointValue : window.outerWidth > breakpointValue;
   }
 
-  get tab() {
-    // Check if the appropriate tag exists already
+  get tabs() {
     return this.shadowRoot.querySelector(`${PfeTabs.tag}[visible-at="large"]`);
   }
 
   get accordion() {
-    // Check if the appropriate tag exists already
     return this.shadowRoot.querySelector(`${PfeAccordion.tag}[visible-at="small"]`);
   }
 
-  get displayTemplate() {
```

Comment:
Removed displayTemplate because it was fetching the same template for both components in the shadowDOM (tabs and accordion) depending on which one was visible. (So tabs were getting the accordion template)